scripts/build.js

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env node
+const execa = require('execa');
+const program = require('commander');
+const path = require('path');
+const remove = require('lodash/remove');
+const EventEmitter = require('events');
+const { Worker } = require('worker_threads');
+
+const myEmitter = new EventEmitter();
+
+let modules;
+let i = 0;
+
+async function retrieveDependencies(_modules) {
+  const rtn = _modules;
+  const packages = await Promise.all(
+    rtn.map(pck => execa
```

Comment:
As `map` creates a copy, why don't you directly use `_.modules`

Reply:
You mean `_modules`